core/src/sigverify_stage.rs

```diff
@@ -177,6 +200,7 @@ impl SigVerifyStage {
     ) -> Result<()> {
         let (mut batches, num_packets, recv_duration) = streamer::recv_packet_batches(recvr)?;
 
+        let mut discard_time = Measure::start("sigverify_discard_time");
```

Comment:
Let's move this to just above `if num_packets > MAX_SIGVERIFY_BATCH {`.  `batches.len()` is O(1), but I think the scheduler could skew us if we get preemted in `timing::timestamp()`.